code/__DEFINES/_globals.dm

```diff
@@ -20,11 +21,16 @@
 #define GLOBAL_PROTECT(X)
 #endif
 
-/// Standard BYOND global, do not use
-#define GLOBAL_REAL_VAR(X) var/global/##X
+/// Standard BYOND global, seriously do not use without an earthshakingly good reason
+#define GLOBAL_REAL_VAR(X) var/global/##X;
 
-/// Standard typed BYOND global, do not use
-#define GLOBAL_REAL(X, Typepath) var/global##Typepath/##X
+// ▲▼ I added the semi-colons because people kept using these to fuck with init order. See /code/game/world.dm for that - Dominion/Cyberboss
```

Comment:
comment signing

Reply:
you know we dont do this

Reply:
we also don't do historical comments like these really

Reply:
```suggestion
```